src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -2071,3 +2075,161 @@ def managed_db_restore(
         managed_instance_name=target_managed_instance_name,
         resource_group_name=target_resource_group_name,
         parameters=kwargs)
+
+###############################################
+#              sql failover-group             #
+###############################################
+
+
+# pylint: disable=too-few-public-methods
+class FailoverPolicyType(Enum):
+    automatic = 'Automatic'
+    manual = 'Manual'
+
+
+def failover_group_create(
+        cmd,
+        client,
+        resource_group_name,
+        server_name,
+        failover_group_name,
+        partner_server,
+        partner_resource_group=None,
+        failover_policy=FailoverPolicyType.automatic.value,
+        grace_period=1,
+        add_db=None):
+
```

Comment:
please add doc comments e.g. '''Creates a failover group.''' to all new classes & methods